### DISCUSSION TITLE
How to start as a service in Ubuntu

### DISCUSSION THREAD

**Discussion by kerosumi** [ArcadeData/arcadedb#445] [General]
Hi Guys,
How can I start arcadedb as a service in ubuntu20.4? Orientdb had instruction and a file to use with systemd. ANY SUGGESTIONS?

**Comment by lvca** [comment]
We didn't provide instructions for ArcadeDB, but I guess the same OrientDB file should work for ArcadeDB with little changes. In case you already did it, could you please post the file if worked? We'd like to include it in our docs/distro. Thanks in advance.

**Reply by kerosumi** [reply]
I did  try for a little bit but unable to make it work.

**Reply by kovacm** [reply]
Did you manage to start ArcadeDB as a service?

As I see, problem is that 

`$ export ARCADEDB_HOME=/path/to/arcadedb`

is not permanent after reboot (at least in Debian 12) so when you try to run ArcadeDB as service it will not have correct path...

**Reply by kovacm** [reply]
OK, now I am really puzzled. I put in "/etc/environment" line "ARCADEDB_HOME=/home/arcadedb-24.6.1".

After reboot if I run "printenv" I will get: "ARCADEDB_HOME=/home/arcadedb-24.6.1"

So far, so good. (or not? :D)

If I try to run ArcadeDB from /root folder like "/home/arcadedb-24.6.1/bin/server.sh" ArcadeDB will create folders "database" and "logs" in "/root". It seems that "environment variable" is not take into consideration... :)

**Comment by tolgaulas** [comment]
@kerosumi this is mine and it work at the moment:
```
[Unit]
Description=Arcadedb service
After=network.target

[Service]
Type=simple
ExecStart=/arcadedb/bin/server.sh
WorkingDirectory=/arcadedb
Restart=on-failure
RestartSec=5
User=root
LimitNOFILE=65536

[Install]
WantedBy=multi-user.target
```
